sdk/core/azure-core/test/ut/transport_policy_options.cpp

```diff
@@ -524,7 +524,13 @@ namespace Azure { namespace Core { namespace Test {
 #endif
   }
 
+// This test started to fail consistently in CI on Linux.
+// It should be fixed, but to unblock CI, there is a workaround below.
+#if !defined(AZ_PLATFORM_LINUX)
   TEST_F(TransportAdapterOptions, MultipleCrlOperations)
+#else
+  TEST_F(TransportAdapterOptions, MultipleCrlOperations_LIVEONLY_)
+#endif
```

Comment:
Please file an issue to track the CI failure. Is it rare and intermittent (where a re-run succeeds), or does it happen quite frequently?

Reply:
I saw it yesterday and since then, it is consistent, does not fail on rerun.

Reply:
If the issue is with example.com, it is surprising that it only fails on linux.

Reply:
#4237

Reply:
I think it is an issue with the connection pool, not with any site.